modules/rosa-sts-creating-cluster.adoc

```diff
@@ -14,13 +21,25 @@ link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Share
 
 .Procedure
 
-When creating your cluster, use the following default options to configure your networking IP ranges. For more information when using manual mode, use `rosa create cluster --help | grep cidr`. In interactive mode, you are prompted for the settings.
-
-* Machine CIDR: 10.0.0.0/16
-* Service CIDR: 172.30.0.0/16
-* Pod CIDR: 10.128.0.0/14
+. Create the required account-wide roles and policies, including Operator policies:
++
+[source,terminal]
+----
+$ rosa create account-roles --version 4.8 <1>
+----
+<1> You must specify an OpenShift release by using the `--version` option.
++
+You can select from the following modes in the interactive prompt:
++
+* `auto`: The account roles and and policies are created directly using the current AWS account.
+* `manual`: The role and policy JSON files are saved in the current directory. The command output includes the `aws` CLI commands required to create the roles.
++
+[NOTE]
+====
+Alternatively, you can specify the mode by using the `--mode auto -y` or `--mode manual` CLI arguments.
+====
 
-. You can create a cluster by specifying the specifying the custom settings shown below or by using the interactive mode. To view other options when creating a cluster, enter `rosa create cluster --help`.
+. Create a cluster by specifying the custom settings shown below or by using the interactive mode. To view other options when creating a cluster, enter `rosa create cluster --help`.
 +
 Creating a cluster can take up to 40 minutes.
 +
```

Comment:
This block (and the `[NOTE]` below) belongs to the `rosa create cluster` command.

Reply:
@vkareh thank you for your further review! Can you please expand on this comment? The block that this relates to is part of the step that provides options for the create cluster commands. In the preview, the section includes the above text and then two sub-bullet points that provide the custom command and interactive command options for `rosa cluster create`. Would you like it to be positioned a different way? Thanks.

Reply:
Okay I see it now, my bad. Looking at the adoc code, it seemed as though it was in the previous section, but looking at the preview everything looks in the right place :+1:

Reply:
Great, thank you for confirming that.